Revision Mods/style.css

```diff
@@ -0,0 +1,123 @@
+
+
+.header-container {
+    background: #469654;
+    display: flex;
+    justify-content: center;
+}
+
+.body-margin {
+    margin-top: 30px;
+}
+
+.page-padding {  
+    margin: 50px 300px;
+    max-width: 800px;
+   
+} 
+
+.main-content {
+    font-family: Arial, Helvetica, sans-serif;
+    font-size: 18px; 
+    color: black
+}
+
+.name-font {
+    font-family: Arial, Helvetica, sans-serif;
+    font-size: 30px; 
+    margin: 0;
+    padding: 0;
+
+}
+
+hr {
+    height: 0px;
+    border-top: 3px solid;
+}
+
+/* Parent container for Leo and Mike */
+.right-container {
+    padding: 20px;
+    display: flex;
+    justify-content: center;
+}
+
+
+/* Parent container for Don and Raph */
+.left-container {
+    padding: 20px;
+    display: flex;
+    justify-content: center;
+    flex-direction: row-reverse;
+}
+
+
+/*Formatting for Leo and Mike*/
+.right-child-element {
+    
+    width: 800px;
+    display: flex;
+    justify-content: space-between;
+}
+
+/*Formatting for Don and Raph*/
+.left-child-element {
+    
+    width: 800px;
+    display: flex;
+    justify-content: space-between;
+    flex-direction: row-reverse;   
+}
+
+
+/*Puts turtle name. hr tag, and bio in a column*/
+.stack-bio {
+  
+    width: 800px;
+    display: flex;
+    flex-direction: column;
+}
+
+
+/*Rounded-image-left and -right create rounded image corners*/
+.rounded-image-right {
```

Comment:
Looks pretty good, but you could consolidate the `rounded-image-right` and `rounded-image-left` classes a little bit. For example you could create `.rounded-image` that has the border-radius, width and height elements on them. and then create  something like `.image-left` to add the margin-right and `.image-right` to add the margin-right.

This makes it so your rounded image class is more portable and your alignment classes are reusable